rclrs/src/node/builder.rs

```diff
@@ -159,11 +274,38 @@ impl NodeBuilder {
         // SAFETY: No preconditions for this function.
         let mut node_handle = unsafe { rcl_get_zero_initialized_node() };
```

Comment:
Can you move this line further down, close to `rcl_node_init()` where it is used?